pkg/errctx/center.go

```diff
@@ -0,0 +1,40 @@
+package errctx
+
+import (
+	"context"
+
+	"go.uber.org/atomic"
+)
+
+type ErrCenter struct {
```

Comment:
prefer the name `FirstError`

Reply:
I think `FirstError` is too simple to describe the object's functionality. Don't you think so?

Reply:
From `ErrCenter` I can not know its functionality, maybe left some comments